tests/core/pyspec/eth2spec/test/electra/block_processing/test_process_deposit_request.py

```diff
@@ -2,292 +2,139 @@
     validator_index = 0
-    amount = spec.MAX_EFFECTIVE_BALANCE // 4
+    amount = spec.MIN_ACTIVATION_BALANCE // 4
     deposit_request = prepare_deposit_request(spec, validator_index, amount, signed=True)
 
-    state.balances[validator_index] = spec.MAX_EFFECTIVE_BALANCE
-    state.validators[validator_index].effective_balance = spec.MAX_EFFECTIVE_BALANCE
+    state.balances[validator_index] = spec.MIN_ACTIVATION_BALANCE
+    state.validators[validator_index].effective_balance = spec.MIN_ACTIVATION_BALANCE
 
     yield from run_deposit_request_processing(spec, state, deposit_request, validator_index)
 
-    deposits_len = len(state.pending_balance_deposits)
-    assert state.pending_balance_deposits[deposits_len - 1].amount == amount
-    assert state.validators[validator_index].effective_balance == spec.MAX_EFFECTIVE_BALANCE
-
 
 @with_electra_and_later
 @spec_state_test
-def test_top_up__less_effective_balance(spec, state):
+def test_process_deposit_request_top_up_max_effective_balance_compounding(spec, state):
     validator_index = 0
-    amount = spec.MAX_EFFECTIVE_BALANCE // 4
-    deposit_request = prepare_deposit_request(spec, validator_index, amount, signed=True)
-
-    initial_balance = spec.MAX_EFFECTIVE_BALANCE - 1000
-    initial_effective_balance = spec.MAX_EFFECTIVE_BALANCE - spec.EFFECTIVE_BALANCE_INCREMENT
-    state.balances[validator_index] = initial_balance
-    state.validators[validator_index].effective_balance = initial_effective_balance
-
-    yield from run_deposit_request_processing(spec, state, deposit_request, validator_index)
-
-    deposits_len = len(state.pending_balance_deposits)
-    assert state.pending_balance_deposits[deposits_len - 1].amount == amount
-    # unchanged effective balance
-    assert state.validators[validator_index].effective_balance == initial_effective_balance
-
-
-@with_electra_and_later
-@spec_state_test
-def test_top_up__zero_balance(spec, state):
-    validator_index = 0
-    amount = spec.MAX_EFFECTIVE_BALANCE // 4
-    deposit_request = prepare_deposit_request(spec, validator_index, amount, signed=True)
-
-    initial_balance = 0
-    initial_effective_balance = 0
-    state.balances[validator_index] = initial_balance
-    state.validators[validator_index].effective_balance = initial_effective_balance
-
-    yield from run_deposit_request_processing(spec, state, deposit_request, validator_index)
-
-    deposits_len = len(state.pending_balance_deposits)
-    assert state.pending_balance_deposits[deposits_len - 1].amount == amount
-    # unchanged effective balance
-    assert state.validators[validator_index].effective_balance == initial_effective_balance
-
-
-@with_electra_and_later
-@spec_state_test
-@always_bls
-def test_incorrect_sig_top_up(spec, state):
-    validator_index = 0
-    amount = spec.MAX_EFFECTIVE_BALANCE // 4
-    deposit_request = prepare_deposit_request(spec, validator_index, amount)
-
-    # invalid signatures, in top-ups, are allowed!
```

Comment:
This insightful comment is no longer in `test_process_deposit_request_top_up_invalid_sig`.

Is this still accurate?

Reply:
It is a bit tricky. This is still true, but the test that checks that is `test_apply_pending_deposit_incorrect_sig_top_up` as there is no sig validations on deposit request processing. Deposit request tests ensures that pending deposits are created with invalid deposit request data of different kind, i.e. all validations are _deferred_ to `apply_pending_deposit`.